### PR TITLE
Register new queries available in InfluxDB - Holt Winters

### PR DESCRIPTION
**Pull request by rikkuness**
For correct data fill on holt winters predictions, the time function needed modification to allow a 2nd param for offset, as per [documentation].(https://docs.influxdata.com/influxdb/v1.0/query_language/data_exploration/#advanced-group-by-time-syntax)

I couldn't see a way to add a param to time() as optional though, from my testing having both interval and offset set to $interval or 'auto' works perfectly well and all the grunt tests pass.

#5619 refers

### CODE REVIEW

File: pkg/tsdb/influxdb/query_part.go [hunk 1]
```diff
@@ -50,7 +59,7 @@ func init() {
 	renders["stddev"] = QueryDefinition{Renderer: functionRenderer}
 	renders["time"] = QueryDefinition{
 		Renderer: functionRenderer,
-		Params:   []DefinitionParameters{{Name: "interval", Type: "time"}},
+		Params:   []DefinitionParameters{{Name: "interval", Type: "time"}, {Name: "offset", Type: "time"}},
```
Comment (torkelo) [inline pkg/tsdb/influxdb/query_part.go]: Afraid this is a breaking change? What of people running on older versions of inflxudb where the offset parameter does not exist?

**Comment by torkelo** [comment]
This looks good, but the change to group by time looks like a breaking change, so not sure I can merge it.

**Comment by rikkuness** [comment]
Yeah I agree, I'm not sure the best course of action, the 2nd param to time needs to be optional but I'm not sure that's possible in the current UI unless I'm missing something, maybe a separate issue needs raising to cover optional parameters?

**Comment by skuxy** [comment]
Hi, is there any possibility of adding time offset in near~ish future?

**Comment by torkelo** [comment]
what is the use case for interval offset? can we remove that from this PR as it is a breaking change. It seems like an edge case case that we can solve by using the raw query text editor.

**Comment by rikkuness** [comment]
I didn't get a lot of useful data out of the predictions without the offset/interval stuff, it's been a while since I touched this code so I can't remember off hand, but it seemed to be needed. 

Good to see it merged anyway :) Thanks.

**Pull request merged**